crates/graphql/src/schema.rs

```diff
@@ -999,6 +1153,89 @@ impl BatchFn<String, Vec<AuctionHouse>> for AuctionHouseBatcher {
     }
 }
 
+struct BidReceiptBatcher {
+    db_pool: Arc<Pool>,
+}
+
+#[async_trait]
+impl BatchFn<String, Vec<BidReceipt>> for BidReceiptBatcher {
+    async fn load(&mut self, metadata_addresses: &[String]) -> HashMap<String, Vec<BidReceipt>> {
+        let conn = self.db_pool.get().unwrap();
+        let mut hash_map = HashMap::new();
+
+        for address in metadata_addresses {
+            hash_map.insert(address.clone(), Vec::new());
+        }
+
+        let rows: Vec<models::BidReceipt> = bid_receipts::table
+            .inner_join(metadatas::table.on(metadatas::address.eq(bid_receipts::metadata)))
+            .inner_join(
+                token_accounts::table.on(token_accounts::mint_address.eq(metadatas::mint_address)),
```

Comment:
maybe we don't need to check token amount and remove this extra join.

Reply:
yeah. we are going to use ah public bids which are associated to the nft and not the owner. If we did private sales then only want to return if the bid is only valid if its owned by the person it was set for.